docs/conf.py

```diff
@@ -26,7 +26,7 @@
 author = "HPy Collective"
 
 # The full version, including alpha/beta/rc tags
-release = "0.0.4"
+release = "0.9"
```

Comment:
jump in version?

Reply:
Yes. We used version `0.0.x` to indicate that we are in an early (alpha) phase. We now think that we left the _alpha_ phase and want to indicate this by jumping to `0.x`. We actually discussed if this release should be `1.0` but we wanted to leave some _space_.